packages/entities/entities-plugins/src/components/PluginConfigCard.vue

```diff
@@ -132,6 +169,7 @@ const emit = defineEmits<{
   (e: 'error:fetch-schema', error: AxiosError): void,
   (e: 'fetch:success', data: Record<string, any>): void,
   (e: 'copy:success', data: Record<string, any>): void,
+  (e: 'navigation-click', data: string, direction: 'route' | 'consumer' | 'consumer_group' | 'service'): void
```

Comment:
Also please document this event [here](https://github.com/Kong/public-ui-components/blob/main/packages/entities/entities-plugins/docs/plugin-config-card.md#events).